pkg/oci/pusher/pusher.go

```diff
@@ -291,15 +295,20 @@ func (p *Pusher) packManifest(ctx context.Context, fileStore *file.Store,
 	// Now we can create manifest, using the Config descriptor and principal Layer descriptor.
 	// In case annotation source is passed, we put it in the ManifestAnnotations.
 	var packOptions oras.PackOptions
+
+	// Currently, Manifests are not pushed as OCI Artifact Manifest.
+	// Always pushed as OCI Image Manifest.
+	packImageManifest := true
```

Comment:
Nice that we can have this option so that we can plan when to switch to OCI Artifact Manifests later (maybe if we will consider using the `subject` fields, potentially for signing).

Reply:
This way we should expect a consistent behaviour with `packImageManifest=true` and switch when we want 😍

Reply:
I agree.
But it is a bit difficult to me, designing and testing compatibility mode for ORAS,
Only bump will done here, and add this comment instead.

https://github.com/oras-project/oras/blob/v1.0.0-rc.1/docs/proposals/compatibility-mode.md